README.md

```diff
@@ -1 +1,61 @@
-# SENG 499 - Company 2 - Algorithm 1 
\ No newline at end of file
+# SENG 499 - Company 2 - Algorithm 1 
+
+This repository contains a package, [Course-Scheduler](https://pypi.org/project/coursescheduler/), for
+that returns a course schedule for three terms of Software Engineering courses at UVic. The Course Scheduler uses the 
+Constraint Satisfaction Problem (CSP) Algorithm to determine the hard constraint of the schedule, and produces a schedule 
+that adheres to the constraints. The scheduler also takes into account soft constraints for professor preferences, 
+however, these constraints are not always fulfilled. 
+
+This package was created for the Capstone SENG499 class: Company 2 Course Scheduler Project. 
+Authored by the Company2-Algorithm1 sub-team.
+
+## Install from PyPi Using PIP
+Install this package into your environment from PyPi using `pip`. 
+
+```bash
+$ pip install coursescheduler==0.0.1
```

Comment:
This version number could be removed to ensure the latest version is always installed
```suggestion
$ pip install coursescheduler
```

Reply:
Thank you for catching that!